DTS-STN/Alpha-Site
# Fix focus on feedback widget

# Description

[Focus Indicator Disappears While Tabbing](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-606)

Fixed the focus indicator on the feedback widget on the close button and the email "experience@servicecanada.gc.ca"

## Test Instructions

1. Run the code locally
2. Check the feedback widget to see if the close button and the email "experience@servicecanada.gc.ca" focus on tab

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)
